exchanges/exchange.go

```diff
@@ -1143,18 +1143,23 @@ func (b *Base) FormatExchangeKlineInterval(in kline.Interval) string {
 // ValidateKline confirms that the requested pair, asset & interval are
 // supported and/or enabled by the requested exchange.
 func (b *Base) ValidateKline(pair currency.Pair, a asset.Item, interval kline.Interval) error {
```

Comment:
Unexport as its not part of IBOTExchange

suggestion: Change name to something like `VerifyKlineParameters` as we are not validating a kline directly